### PR TITLE
Fix various issues to get unit tests working for VirtualMachineTests

### PR DESCRIPTION
**Pull request by MatthewJohn**
* Temporarily disable other unit tests
 * Update daemon configuration to handle a maximum of 128 connections and to queue connections after this, rather than close them.
 * Add base class for unit tests, containing
 * Fix issues with VM duplication and cloning
 * Expose/create various supporting methods for unit tests

### CODE REVIEW

**Comment by MatthewJohn** [comment]
Have now fixed all unit tests, apart from auth tests, which have been disabled until the implementation of #182

Output from tests:
    
    ----------------------------------------------------------------------
    Ran 52 tests in 336.457s
    
    OK (skipped=2)

**Comment by itdevreviewboard** [comment]
Build triggered. sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build started sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build finished.

File: source/usr/lib/mcvirt/test/unit_test_bootstrap.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+from mcvirt.test.update_tests import UpdateTests
+from mcvirt.test.virtual_machine.online_migrate_tests import OnlineMigrateTests
+from mcvirt.rpc.rpc_daemon import RpcNSMixinDaemon
+
+
+class UnitTestBootstrap(object):
+    """Bootstrap daemon with unit tests"""
+
+    def __init__(self):
+      """Create dameon and test suite objects"""
```
Comment (joesingo) [inline source/usr/lib/mcvirt/test/unit_test_bootstrap.py]: Needs 4-space indentation

**Comment by joesingo** [comment]
After discussion with @MatthewJohn, everything looks good to me :shipit:

**Comment by itdevreviewboard** [comment]
Build triggered. sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build started sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build finished.

**Comment by itdevreviewboard** [comment]
Build started sha1 is merged.

**Comment by itdevreviewboard** [comment]
Build finished.

**Pull request merged**